### ISSUE TITLE
Markdown parsing is not working

### ISSUE THREAD

**Issue by jay-pee** [glfharris/ankidown#1]
Thank you for your work in this add on. :) I hope I find the time to contribute.

But to the topic:

Formatting doesn't work or am I doing something wrong?

1. Code blocks are not formatted

For Example:
```` 
```python
    import sys
    sys.path
```
````

is not rendered to
```python
    import sys
    sys.path
```

2. List formatting doesn't work
3. Bold and Italic doesn't work.

Had no time to look in the code. But do you render markdown to html and then import it to anki? For me it looks like the plain text is just imported...

**Comment by glfharris** [comment]
Hi!

Currently there's an option in the config file to decide whether to import plain text, or to run it through a basic md->html converter.
Currently the formatting code is very primitive (literally one line in ankidown/note.py), but for me it's worked on lists and things.

**Comment by jay-pee** [comment]
When i look at my config file i have the option:

       ```"format": "markdown",```

so it should convert to markdown.

However when I press "preview" it just copies the raw text like so:
![Capture](https://user-images.githubusercontent.com/9380980/60737203-2c436c80-9f5a-11e9-99de-5d0775acea71.PNG)

**Comment by glfharris** [comment]
Was caught out by pythons string comparisons, has also been fixed in the v0.0.1 version on Ankiweb.
